backend/settings/settings.py

```diff
@@ -16,15 +16,14 @@
 import yaml
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
+from dotenv import load_dotenv
```

Comment:
This import could be moved above the comment about `BASE_DIR`.

Reply:
thanks, updated.